libcrux-ml-dsa/src/simd/portable/ntt.rs

```diff
@@ -1,67 +1,77 @@
-use super::arithmetic::*;
-use crate::simd::portable::PortableSIMDUnit;
+use super::arithmetic;
+use crate::simd::{
+    portable::PortableSIMDUnit,
+    traits::{
+        montgomery_multiply_by_fer, COEFFICIENTS_IN_SIMD_UNIT, SIMD_UNITS_IN_RING_ELEMENT,
+        ZETAS_TIMES_MONTGOMERY_R,
+    },
+};
 
 #[inline(always)]
-pub fn ntt_at_layer_0(
+pub fn simd_unit_ntt_at_layer_0(
     mut simd_unit: PortableSIMDUnit,
     zeta0: i32,
     zeta1: i32,
     zeta2: i32,
     zeta3: i32,
 ) -> PortableSIMDUnit {
-    let t = montgomery_multiply_fe_by_fer(simd_unit.coefficients[1], zeta0);
+    let t = arithmetic::montgomery_multiply_fe_by_fer(simd_unit.coefficients[1], zeta0);
```

Comment:
What's the reason to use the path here instead of importing?

Reply:
Ah, I don't see a reason to avoid the import, thanks :+1: